assets/js/blocks/active-filters/utils.js

```diff
@@ -41,26 +41,30 @@ export const formatPriceRange = ( minPrice, maxPrice ) => {
  * @param {string} type Type string.
  * @param {string} name Name string.
  * @param {Function} removeCallback Callback to remove item.
+ * @param {boolean} showLabel Should the label be shown for this item?
```

Comment:
You can specify the default value in JS doc:
```suggestion
 * @param {boolean} [showLabel=true] Should the label be shown for this item?
```

You could also take the chance to mark `[removeCallback]` as optional with square brackets.